lms/static/karma_lms.conf.js

```diff
@@ -27,6 +27,7 @@ var options = {
     // Otherwise Istanbul which is used for coverage tracking will cause tests to not run.
     sourceFiles: [
         {pattern: 'coffee/src/**/!(*spec).js'},
+        {pattern: 'course_bookmarks/**/!(*spec).js'},
```

Comment:
I like the idea of the separate suite for features, so I've created a ticket:

https://openedx.atlassian.net/browse/LEARNER-267